packages/graphql-mesh-server/lib/pipeline.ts

```diff
@@ -90,5 +104,44 @@ export class CodePipelineService extends Construct {
         }),
       ],
     });
+
+    if (props.notificationArn) {
+      const notifier = new NodejsFunction(this, "NotifierLambda", {
+        entry: path.resolve(__dirname, "../assets/handlers/notify-sns.ts"),
```

Comment:
@AdamJHall this would upload the ts file without compilation to js right?

Reply:
CDK auto detects that it is Typescript and transpiles it itself based on your global tsconfig

Reply:
:open_mouth: Does it also bundle and transpile? If the function has a dependency or a package will it be included in the function? (excluding `aws-sdk` because these are available in the runtime already)

Reply:
I believe so but not 100%, it just uses esbuild under the hood so it should.

Reply:
Ahh yes it does: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs-readme.html#install-modules